res/mkn/hip.yaml

```
@@ -0,0 +1,22 @@
+property: 
```

yamllint (1.29.0-1)

[error] trailing spaces

(trailing-spaces)